nano/templates/code_mirror.tmpl

```diff
@@ -0,0 +1,14 @@
+<h1>HELLO</h1>
```

Comment:
I never developed it past making the codemirror window appear and take over the textbox- but it most assuredly works that far.